frontend/src/components/ChangeRequests/ChangeRequestsList/ChangeRequestsList.jsx

```diff
@@ -3,37 +3,48 @@ import PropTypes from "prop-types";
 import { useGetChangeRequestsListQuery } from "../../../api/opsAPI";
 import BudgetChangeReviewCard from "../BudgetChangeReviewCard";
 import StatusChangeReviewCard from "../StatusChangeReviewCard";
+import { useSelector } from "react-redux";
 
 /**
  * Change Requests List component.
  * @component
+ * @typedef {import("../ChangeRequestsTypes").ChangeRequest} ChangeRequest
  * @param {Object} props
  * @param {Function} props.handleReviewChangeRequest - Function to handle review of change requests
  * @returns {JSX.Element} - The rendered component
  */
 function ChangeRequestsList({ handleReviewChangeRequest }) {
+    const userDivisionId = useSelector((state) => state.auth?.activeUser?.division) ?? -1;
     const {
         data: changeRequests,
         isLoading: loadingChangeRequests,
         isError: errorChangeRequests
     } = useGetChangeRequestsListQuery({ refetchOnMountOrArgChange: true });
+
     if (loadingChangeRequests) {
         return <h1>Loading...</h1>;
     }
     if (errorChangeRequests) {
         return <h1>Oops, an error occurred</h1>;
     }
 
-    /**
-     *  @typedef {import('../ChangeRequestsTypes').ChangeRequest} ChangeRequest
-     *  @type {ChangeRequest[]}
-     */
-    return changeRequests.length > 0 ? (
+    const changeRequestManagingIds = Array.isArray(changeRequests)
+        ? changeRequests.map(
+              /** @param {ChangeRequest} changeRequest */ (changeRequest) => changeRequest.managing_division_id
+          )
+        : [];
+    const changeRequestsForUser = Array.isArray(changeRequests)
+        ? changeRequests.filter(
+              /** @param {ChangeRequest} changeRequest */
+              (changeRequest) => changeRequest.managing_division_id === userDivisionId
+          )
+        : [];
+    console.log({ userDivisionId, changeRequestManagingIds, changeRequestsForUser });
```

Comment:
Do you need the console.log statement still? Or was this just for testing.

Reply:
thank you, I will remove it